docs/multi-cluster-ingress.md

```diff
@@ -27,55 +27,45 @@ We use [multi-cluster-ingress-nginx](https://github.com/karmada-io/multi-cluster
 
 ```shell
 # for HTTPS
-git clone https://github.com/karmada-io/multi-cluster-ingress-nginx.git
+$ git clone https://github.com/karmada-io/multi-cluster-ingress-nginx.git
```

Comment:
Is this format standardized?

Reply:
Here I read other documents, in other documents have used this format（https://github.com/karmada-io/karmada/blob/master/docs/descheduler.md）, so here for reference